### PR TITLE
update footer for site

### PR DESCRIPTION
**Pull request by Ilyaobr**

### CODE REVIEW

File: styles/footer.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+}
+
+@media screen and (min-width: 600px) {
+  .footer {
+    display: flex;
```
Comment (velmyk) [inline styles/footer.css]: o not duplicate styles inside media query

File: styles/footer.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    max-width: 1400px;
+    margin: auto;
+    padding: 48px;
+    justify-content: space-between;
+    flex-direction: row;
+  }
+}
+
+.under-label-left {
+  margin-left: 1vh;
```
Comment (velmyk) [inline styles/footer.css]: Should it be in `px`?

File: styles/footer.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.footer-column-for-text {
+  margin: 2vh;
+  margin-left: 0;
+}
+
+@media screen and (min-width: 600px) {
+  .under-label {
```
Comment (velmyk) [inline styles/footer.css:29]: join this 2 media queries

File: styles/footer.css [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    align-items: center;
+    flex-direction: row;
+    max-width: 1400px;
+    margin: auto;
+  }
+}
+
+.company-name {
+  font-family: "Work Sans", sans-serif;
+  margin-top: -20px;
```
Comment (velmyk) [inline styles/footer.css:47]: Don't use regative margins

File: styles/footer.css [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+  font-style: normal;
```
Comment (velmyk) [inline styles/footer.css:48]: Is it needed?

File: styles/footer.css [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 26px;
+  color: rgba(43, 41, 45, 0.6);
+}
+
+@media screen and (min-width: 600px) {
+  .company-name {
+    margin-left: 60px;
+    margin-top: -25px;
```
Comment (velmyk) [inline styles/footer.css:58]: Don't use neg margins

File: styles/footer.css [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+  margin-top: 20px;
+  font-family: "Work Sans", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 26px;
+  color: rgba(43, 41, 45, 0.6);
+}
+
+@media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
```
Comment (velmyk) [inline styles/footer.css:137]: Are you sure about this condition?

**Review by velmyk — CHANGES_REQUESTED** [review]

**Review by velmyk — APPROVED** [review]

File: styles/footer.css (hunk 1 above)
Comment (Ilyaobr) [reply styles/footer.css]: done

**Pull request merged**